examples/scheme-client/index.ts

```diff
@@ -1,5 +1,6 @@
 process.env.YDB_SDK_PRETTY_LOGS = '1';
 
+import {ModifyPermissionsSettings} from 'ydb-sdk';
```

Comment:
Suggestions on commit message
1. What is breaking change here?
2. If that's refactoring, use 'refactor' type, not 'chore'.

Reply:
I temporarily revert this changes, because I want to rewrite scheme client methods much cleaner, so it's not actual now.